frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -4,6 +4,7 @@ import {
     IconDashboard,
     IconDatabase,
     IconDecisionTree,
+    IconFeatures,
```

Comment:
[nit] not your fault at all, but it's kinda annoying how our feature icon and our AI icon are both iterations of the sparkle icon.  Feels like we could come up with something better here for our features _product_, but not sure if it's worth the effort yet.

Reply:
Agree on both fronts; let's get something new but asset swapping can wait til real launch!